content/applications/finance/fiscal_localizations/brazil.rst

```diff
@@ -0,0 +1,315 @@
+======
+Brazil
+======
+
+.. |IAP| replace:: :abbr:`IAP (In-app-purchase)`
+.. |SO| replace:: :abbr:`SO (Sales order)`
+
+Introduction
+============
+
+With the Brazilian localization you can automatically compute sales taxes for goods using AvaTax
+(Avalara) through API calls, also configure taxes for services.
+
+For the goods tax computation part, you need to configure the :ref:`contacts <brazil/contacts>`,
+:ref:`company <brazil/company>`, :ref:`products <brazil/products>`, and :ref:`create an account in
+Avatax <brazil/avatax-account>` from the Odoo general settings.
+
+For the services taxes, you can create and configure them from Odoo directly without computing them
+with AvaTax.
+
+The localization also includes taxes and a chart of accounts template that can be modified if
+needed.
+
+Configuration
+=============
+
+Modules installation
+--------------------
+
+:ref:`Install <general/install>` the following modules to get all the features of the Brazilian
+localization:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Brazilian - Accounting`
+     - `l10n_br`
+     - Default :ref:`fiscal localization package <fiscal_localizations/packages>` - adds accounting
+       characteristics for the Brazilian localization, which represent the minimum configuration
+       required for a company to operate in Brazil. The module's installation automatically loads:
+       the chart of accounts, taxes, and required fields to properly configure the contact.
+   * - :guilabel:`Brazil - Accounting Reports`
+     - `l10n_br_reports`
+     - Adds a simple tax report that helps check the tax amount per tax group in a given period of
+       time. Also adds the P&L and BS adapted for the Brazilian market.
+   * - :guilabel:`Avatax Brazil`
+     - `l10n_br_avatax`
+     - Add Brazilian tax calculation via Avatax and all necessary fields needed to configure Odoo in
+       order to properly use Avatax and send the needed fiscal information to retrieve the correct
+       taxes.
+   * - :guilabel:`Avatax for SOs in Brazil`
+     - `l10n_br_avatax_sale`
+     - Same as the `l10n_br_avatax` module with the extension to the sales order module.
+
+.. _brazil/company:
+
+Configure your company
+----------------------
+
+To configure your company information, go to the :menuselection:`Contacts` app and search the name
+given to your company.
+
+#. Select the :guilabel:`Company` option at the top of the page. Then, configure the following
+   fields:
+
+   - :guilabel:`Name`
+   - :guilabel:`Address` (add :guilabel:`City`, :guilabel:`State`, :guilabel:`Zip Code`,
+     :guilabel:`Country`)
+   - Tax ID (:guilabel:`CNPJ`)
+   - :guilabel:`IE`
+   - :guilabel:`IM`
+   - :guilabel:`SUFRAMA code` (if applicable)
```

Comment:
```suggestion
   - :guilabel:`IE` (State Registration)
   - :guilabel:`IM` (Municipal Registration)
   - :guilabel:`SUFRAMA code` (Superintendence of the Manaus Free Trade Zone - if applicable)
```

@samueljlieber maybe something like this. Please feel free to improve it if needed